spec/rational_spec.rb

```diff
@@ -1,9 +1,25 @@
 describe SymEngine::Rational do
   context '#initialize' do
```

Comment:
Now this context should be renamed (it is no more describes an `#initialize` method).